docs/keto/sdk/go.md

```diff
@@ -0,0 +1,231 @@
+---
+id: go
+title: Go
+---
+
+import CodeBlock from '@theme/CodeBlock'
+import { useLatestRelease } from '@site/src/hooks'
+
+
+Ory Keto exposes two APIs for integration
+
+- [gRPC](http://localhost:3001/docs/keto/reference/proto-api)
+- [REST](http://localhost:3001/docs/keto/reference/rest-api)
+
+## Installing SDK
+
+Installing gRPC SDK
+
+```mdx-code-block
+
+<CodeBlock className="language-shell">{`go get  github.com/ory/keto/proto/ory/keto/acl/v1alpha1@${useLatestRelease('keto')}`}</CodeBlock>
+
+```
+
+Installing REST SDK
+
+```mdx-code-block
+<CodeBlock className="language-shell">{`go get  github.com/ory/keto-client-go${useLatestRelease('keto')}`}</CodeBlock>
+```
+
+## Configuration
+
```

Comment:
maybe add a small sentence that highlights the main difference in configuration in REST and grpc?

Reply:
I don't know what to add actually. They are completely different protocols with different clients by design.

Reply:
Makes sense yea. A document explaining the difference or when to use each would be good in general. 
But I guess most people will use REST and will know the difference / know when to use grpc